patches/chrome-browser-ui-cocoa-main_menu_builder.mm.patch

```diff
@@ -11,3 +11,11 @@ index b0f66cc832e85b3d9da0b64ed1c3536563b16656..00414aa8a7edc794ba901d0ea92f44db
                  Item(IDS_REOPEN_CLOSED_TABS_MAC)
                      .command_id(IDC_RESTORE_TAB)
                      .remove_if(is_pwa),
+@@ -427,6 +429,7 @@ base::scoped_nsobject<NSMenuItem> BuildHelpMenu(
+ #if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+             Item(IDS_FEEDBACK_MAC).command_id(IDC_FEEDBACK),
+ #endif
++                Item(IDS_REPORT_BROKEN_SITE_MAC).command_id(IDC_SHOW_BRAVE_WEBCOMPAT_REPORTER),
```

Comment:
why are we putting this in the app menu where it will only show up on macos? In either case menu changes should go in `BraveAppMenuModel::InsertBraveMenuItems`

Reply:
cc @alexwykoff for rationale (I am not a Mac user myself)

Did not know about `BraveAppMenuModel`, sounds good

Reply:
the app menu model will allow you to add it to help for every OS in a single place so you don't have to worry about the details of win, linux, mac help menus

Reply:
From browsing around `app/generated_resources.grd`, it doesn't look like there's any kind of cross-platform "Help" menu. There is a platform-specific help menu option for both Mac and Linux, but not Windows. Unless there's another idea, I'll add some platform `#ifdef`s to `InsertBraveMenuItems` and put the option next to both the Mac and Linux help menus.